huggingface_guess/detection.py

```diff
@@ -144,7 +144,13 @@ def detect_unet_config(state_dict, key_prefix):
         dit_config["axes_dim"] = [16, 56, 56]
         dit_config["theta"] = 10000
         dit_config["qkv_bias"] = True
-        dit_config["guidance_embed"] = "{}guidance_in.in_layer.weight".format(key_prefix) in state_dict_keys
+        if "{}distilled_guidance_layer.layers.0.in_layer.bias".format(key_prefix) in state_dict_keys:
```

Comment:
checking the presence of the distilled guidance layer which is the main structural change from Flux to Chroma